### ISSUE TITLE
MotionControllerTest gaze does not highlight cube

### ISSUE THREAD

**Issue by erichfrazer** [Microsoft/MixedRealityToolkit-Unity#1264]
changing the cube's shader to "Standard", and changing OnFocusEnter/Leave function's calls to EnableKeyword("_EMISSION") works, but what is currently there doesn't seem to highlight anything!

**Comment by StephenHodgson** [comment]
While we do our best not to break them, [Example scenes can, and usually become depreciated](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/master/Assets/HoloToolkit-Examples#when-considering-an-example).

> Examples that are not maintained regularly will be deprecated.

**Comment by erichfrazer** [comment]
it's kind of important, for a new toolkit & hardware, to keep the samples/examples & the code up to date. You guys are monkeying with the motion controller SDK, but the example scenes don't seem to work with the motion controllers (can't use the Grab example to grab anything, can't highlight the cube w/ my gaze, etc). For motion controllers especially, might be nice to ensure at least one exhaustive motion controller sample remains functional?

I'd fix up the Grabber scene sample, but I can't figure it out. Too many things are going on in the Input stack for controllers, I can't follow the thread of "who gets what signal, and when", even though I've read the docs & seen the pictures.

**Comment by keveleigh** [comment]
> Examples that are not maintained regularly will be deprecated.

This rule should be reevaluated. In this case you're applying it to something that used to be a Test scene, which was a completely different concept than the type of scene that rule was written about. Example scenes were examples added by the community which showed off solving a specific problem, and we decided they needed to be kept up to date by the contributor. Test scenes were smaller, showing off a single Toolkit component and how to use it, and should not be broken. Now that those two concepts have been merged, this rule becomes fuzzier, but, if the scenes showing off how to actually use the Toolkit break, we have a problem
